google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -148,7 +149,7 @@ public Project getProject() {
           return Optional.empty();
         }
         if (runtimeOptional.filter(runtime -> runtime == FlexibleRuntime.CUSTOM).isPresent()
-            && !Files.exists(Paths.get(deploymentConfiguration.getDockerFilePath()))) {
+            && !Files.exists(Paths.get(deploymentConfiguration.getDockerDirectoryPath() + DOCKERFILE))) {
```

Comment:
There should be a better way to do this: like using one of the file api's to check if a file exists within the directory. This avoids the concatenation and potential OS issues (may not work in Windows).